slidez-fe-core/src/services/auth/auth-service.ts

```diff
@@ -4,11 +4,18 @@ import { RegisterDto } from '../../containers/user/dto/RegisterDto'
 import { LogInResult } from '../../containers/user/dto/LogInResult'
 import { SignStatus } from '../../containers/user/enums/sign-status'
 import { LogInResponseDto } from '../../containers/user/dto/LogInResponseDto'
+import { TokenDto } from '../../containers/user/dto/TokenDto'
 
 const JWT = 'jwt'
+const getAuthUrl = (endpoint: string) => {
+    return `http://localhost:5000/auth/${endpoint}`
+}
+const loginWithOAuthGoogleUrl: string =
+    'http://localhost:5000/auth/login/google'
+const registerWithOAuthGoogleUrl: string =
+    'http://localhost:5000/auth/register/google'
```

Comment:
Don't use http://localhost:5000 in code, on staging this route will be different and it won't work